R/verb-summarise.R

```diff
@@ -45,6 +64,24 @@ check_summarise_vars <- function(dots) {
   }
 }
 
+check_groups <- function(.groups) {
```

Comment:
This was fine before, but it seemed like a good example for https://github.com/tidyverse/design/issues/124